src/java/org/apache/cassandra/service/accord/AccordCallback.java

```diff
@@ -50,7 +51,7 @@ public void onFailure(InetAddressAndPort from, RequestFailureReason failureReaso
     {
         logger.debug("Received failure {} from {}", failureReason, from);
         // TODO (now): we should distinguish timeout failures with some placeholder Exception
-        callback.onFailure(EndpointMapping.endpointToId(from), new RuntimeException(failureReason.toString()));
+        callback.onFailure(EndpointMapping.endpointToId(from), failureReason == RequestFailureReason.TIMEOUT ? new Timeout(null, null) : new RuntimeException(failureReason.toString()));
```

Comment:
just spent a while making sure internode timeouts trigger user timeouts... so had to fix here!